### DISCUSSION TITLE
Can't input with fcitx5 on Kubuntu 23.10

### DISCUSSION THREAD

**Discussion by tjw123hh** [cpeditor/cpeditor-cn#35] [提问] [answered]
无法用fcitx5输入法输入中文（Kubuntu 23.10）

**Comment by ouuan** [comment]
其他 Qt 应用呢

**Reply by tjw123hh** [reply]
其他的可以

**Comment by ouuan** [comment]
我 Arch Linux 可以用 fcitx5，不太清楚你是什么问题

**Comment by tjw123hh** [comment]
https://github.com/cpeditor/cpeditor-cn/assets/56749271/458acc0c-0fd3-47d4-876d-a958458b0339

**Reply by ouuan** [reply]
Firefox 不是 Qt 应用..

**Comment by ouuan** [comment]
试试设置这些环境变量: https://wiki.archlinux.org/title/Fcitx5#Integration

**Reply by tjw123hh** [reply]
还是不行 但我用的KDE 其他Qt的都行

**Reply by tjw123hh** [reply]
![图片](https://github.com/cpeditor/cpeditor-cn/assets/56749271/1638ac95-6de5-410f-9ea4-93669cd7f51e)

**Reply by tjw123hh** [reply]
qt.xkb.compose: failed to create compose table

**Reply by tjw123hh** [reply, marked as answer]
用 Qt 5.15.10 从源代码构建可以了

**Comment by ouuan** [comment]
看起来更应该考虑用系统自带的 Qt 了 https://github.com/cpeditor/cpeditor/issues/1322#issuecomment-1951355373
